test/bats/tests/aws/BasicTestMount.yaml

```diff
@@ -0,0 +1,20 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: basic-test-mount
+spec:
+  serviceAccountName: basic-test-mount-sa
+  containers:
+  - image: nginx
```

Comment:
The test image used for the e2e jobs has been updated to use the k8s busybox test image. Could you update the yaml to use that?

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/b750a0e3bc3de181429fc68dff18b2b59096f34c/test/bats/tests/azure/pod-secrets-store-inline-volume-crd.yaml#L8-L13